plugins/parcelize/parcelize-compiler/parcelize.backend/src/org/jetbrains/kotlin/parcelize/IrParcelSerializerFactory.kt

```diff
@@ -155,6 +155,14 @@ class IrParcelSerializerFactory(private val symbols: AndroidSymbols) {
                 return ulongArraySerializer
         }
 
+        // Special handled library types
+        when (classifierFqName) {
+            "kotlin.time.Duration" ->
+                return wrapNullableSerializerIfNeeded(
+                    irType, IrUnsafeCoerceWrappedSerializer(longSerializer, symbols.kotlinTimeDuration.defaultType, irBuiltIns.longType)
```

Comment:
If we move this into the above `when`, we could define the unsafe coerce wrapper as `kotlinDurationTimeSerializer` to make it a one-liner like the `ULong` case.

Reply:
Done